enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -475,3 +477,46 @@ class SubsidyAccessPolicyCanRedeemElementResponseSerializer(serializers.Serializ
             "List of reasons why each of the enterprise's subsidy access policies are not redeemable, grouped by reason"
         )
     )
+
+
+# pylint: disable=abstract-method
+class SubsidyAccessPolicyAllocateRequestSerializer(serializers.Serializer):
```

Comment:
Wondering if we should also include the ID of the admin who initiated the action?

Reply:
simple-history will do that for us, and you can see the historical records from the django admin detail view of `LearnerContentAssignment` records.